bitcoin/src/blockdata/transaction.rs

```diff
@@ -981,10 +981,19 @@ impl Transaction {
     /// transaction. It is impossible to check if the transaction is first in the block, so this
     /// function checks the structure of the transaction instead - the previous output must be
     /// all-zeros (creates satoshis "out of thin air").
-    pub fn is_coin_base(&self) -> bool {
+    #[doc(alias = "is_coin_base")] // method previously had this name
+    pub fn is_coinbase(&self) -> bool {
         self.input.len() == 1 && self.input[0].previous_output.is_null()
     }
 
+    /// Checks if this is a coinbase transaction.
+    ///
+    /// This method is deprecated, use [is_coinbase] instead.
```

Comment:
We don't need these lines, the `deprecated` attribute adds all the doc markers we need, right?

Reply:
fixing